.circleci/config.yml

```diff
@@ -254,6 +254,18 @@ jobs:
                 rm objects_v0.1.zip
                 cd ..
               fi
+      - run:
+          name: Build Javascript bindings
+          command: |
+              # wait for npm and emscripten install
+              while [[ ! -f ~/emscripten_installed && ! -f ~/npm_deps_installed ]]; do sleep 2; done
+              cd habitat-sim
+              . ~/.bashrc
+              export PATH=$HOME/miniconda/bin:$PATH
+              . activate habitat
+              export EMSCRIPTEN=$HOME/emsdk/fastcomp/emscripten
+              nvm use v11.9.0
```

Comment:
For future that can be good to have versions mentioned somewhere in dependency or requirements specification and re-use it here.

Reply:
Thanks. I will add to documentation in separate PR later.